db/init/00-database.sql

```diff
@@ -234,22 +250,51 @@ CREATE TRIGGER session_delete_session_result BEFORE DELETE
 ON base.session FOR EACH ROW EXECUTE PROCEDURE
 base.delete_children('session_result', 'session_id');
 
+ALTER TABLE base.session ENABLE ROW LEVEL SECURITY;
+
 
 
 /*Create table session result*/
 CREATE TABLE base.session_result (
-    id SERIAL PRIMARY KEY,
-    alert_operator TEXT NOT NULL,
-    alert_threshold FLOAT NOT NULL,
-    nb_records INTEGER NOT NULL,
-    nb_records_alert INTEGER NOT NULL,
-    nb_records_no_alert INTEGER NOT NULL,
-    created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    session_id INTEGER NOT NULL REFERENCES base.session(id)
+    id SERIAL PRIMARY KEY
+  , alert_operator TEXT NOT NULL
+  , alert_threshold FLOAT NOT NULL
+  , nb_records INTEGER NOT NULL
+  , nb_records_alert INTEGER NOT NULL
+  , nb_records_no_alert INTEGER NOT NULL
+  , user_group TEXT NOT NULL
+  , created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+  , session_id INTEGER NOT NULL REFERENCES base.session(id)
 );
 COMMENT ON TABLE base.session_result IS
 'Session results contain a summary of indicators execution.';
 
+ALTER TABLE base.session_result ENABLE ROW LEVEL SECURITY;
+
+
+/*Create table user*/
+CREATE TABLE base.user (
+    id SERIAL PRIMARY KEY
+  , email TEXT NOT NULL UNIQUE
+  , flag_active BOOLEAN DEFAULT TRUE
+  , oauth_type TEXT NOT NULL
+  , access_token TEXT NOT NULL
+  , refresh_token INTEGER NOT NULL
+  , expiry_token INTEGER NOT NULL
+  , created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+  , updated_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+COMMENT ON TABLE base.user IS
+'User contains information about user and his authentication.';
+
+CREATE TRIGGER batch_updated_date BEFORE UPDATE
+ON base.user FOR EACH ROW EXECUTE PROCEDURE
+base.update_updated_date_column();
+
+
+
+/*TODO: create view on pg_roles*/
```

Comment:
TODO done?

Reply:
will be done in a next step

Reply:
@Moeby document this in an issue please.